src/test/java/teammates/test/cases/webapi/PostStudentProfilePictureActionTest.java

```diff
@@ -0,0 +1,55 @@
+package teammates.test.cases.webapi;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.AccountAttributes;
+import teammates.common.util.Const;
+import teammates.ui.webapi.action.JsonResult;
+import teammates.ui.webapi.action.PostStudentProfilePictureAction;
+import teammates.ui.webapi.output.StudentProfilePictureResults;
+
+/**
+ * SUT: {@link PostStudentProfilePictureAction}.
+ */
+public class PostStudentProfilePictureActionTest extends BaseActionTest<PostStudentProfilePictureAction> {
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.STUDENT_PROFILE_PICTURE;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return POST;
+    }
+
+    @Override
+    @Test
+    public void testExecute() throws Exception {
+        AccountAttributes student1 = typicalBundle.accounts.get("student1InCourse1");
+        loginAsStudent(student1.googleId);
+
+        ______TS("Typical case: upload profile picture operation successful");
```

Comment:
Now you should strive for full coverage. The success path is only one of the existing outcomes. How about others like `null` student profile, profile picture too large, profile picture of wrong content type?

Reply:
- [x] `null` student profile picture in request part
- [x]  profile picture too large
- [x] profile picture of wrong content type
- [X] `null` student profile won't be applicable during `POST` because there is this function that creates/update a profile picture (see below). But it is handled in `DELETE`.
```
            logic.updateOrCreateStudentProfile(
                    StudentProfileAttributes.updateOptionsBuilder(userInfo.id)
                            .withPictureKey(pictureKey)
                            .build());
```